packages/block-library/src/query/utils.js

```diff
@@ -214,3 +215,46 @@ export const getTransformedBlocksFromPattern = (
 	}
 	return { newBlocks: clonedBlocks, queryClientIds };
 };
+
+/**
+ * Helper hook that determines if there is an active variation of the block
+ * and if there are available specific patterns for this variation.
+ * If there are, they are going to be suggested to the user in setup and
```

Comment:
Let's clarify that variation specific patterns take over when present, otherwise it might be interpreted as "in addition to".

Reply:
Updated in: https://github.com/WordPress/gutenberg/pull/44197/commits/ee325a5e9b26cbb1ae03440d70f556777a9b28ef